### PR TITLE
Set up API documentation page with hypothes.is embed

### LINKED ISSUE #196: Add documentation page with embedded example file and hypothes.is embed
Prerequisite for #189. Please embed the following resource as example: http://lobid.org/resources/HT013480902

### PR DESCRIPTION
**Pull request by fsteeg**
Will resolve https://github.com/hbz/lobid-resources/issues/196

See http://lobid.org/resources/api

### CODE REVIEW

**Comment by acka47** [comment]
+1 As written in https://github.com/hbz/lobid-resources/issues/197, there will be at least one field hidden in the API response which, thus, shouldn't be documented. Accordingly, the documentation has to be updated as soon as the fields are hidden and/or other changes are made.

**Comment by ChristophEwertowski** [comment]
+1

**Comment by fsteeg** [comment]
> the documentation has to be updated as soon as the fields are hidden and/or other changes are made

The documents are actually fetched when the page is built, so it's always the current version.

**Comment by acka47** [comment]
> The documents are actually fetched when the page is built, so it's always the current version.

Very good.

**Review by dr0i — APPROVED** [review]

File: conf/messages.de [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+api.basics.field.nested = Geschachtelte Feldsuche: "{0}"
+api.basics.pagination = Suche mit Paginierung
+
+api.content_types.header = Inhaltstypen
+api.content_types.default = Standardmäßig liefert dieser Dienst strukturierte API-Antworten (als JSON):
+api.content_types.negotiate = Er unterstützt Content-Negotiation über den Accept-Header für JSON (application/json) oder HTML (text/html):
+api.content_types.override = Der Query-Parameter "format" kann verwendet werden, um den Accept-Header aufzuheben, z.B. zur Anzeige von JSON im Browser:
+
+api.jsonld.header = JSON-LD
+api.jsonld.info = JSON-API-Anfragen liefern JSON-LD (JSON for Linking Data). Um damit zu experimentieren können sie das JSON-LD oder URLs im <a href="{0}">JSON-LD Playground</a> einfügen.
```
Comment (dr0i) [inline conf/messages.de:15]: "JSON for Linked Data"

**Review by dr0i — CHANGES_REQUESTED** [review]

**Comment by dr0i** [comment]
I like the idea of using hypothesis :+1: 
A small thing:
couldn't find where the page header is set. What speaks against "der hbz-Verbundkatalog als Linked Open Data" instead of only "der hbz-Verbundkatalog als Open Data"?

File: conf/messages.de (hunk 1 above)
Comment (fsteeg) [reply conf/messages.de:15]: Not what they call it: http://json-ld.org/

**Comment by fsteeg** [comment]
> couldn't find where the page header is set. What speaks against "der hbz-Verbundkatalog als Linked Open Data" instead of only "der hbz-Verbundkatalog als Open Data"?

That is unrelated to this change. The header is included on every page via the main template and did not change in this pull request. My reason for the wording "der hbz-Verbundkatalog als Open Data im Web" was mainly that "Open" seemed redundant given the "im Web" part. If we want to discuss and change the wording, we should do so in a separate isse.

**Comment by dr0i** [comment]
Re: LOD We can discuss this further on thursday.
+1 for the rest !

**Review by dr0i — APPROVED** [review]

File: conf/messages.de (hunk 1 above)
Comment (dr0i) [reply conf/messages.de:15]: But here: https://en.wikipedia.org/wiki/Jsonld they reference https://www.w3.org/TR/json-ld-api/ which also name them "linked".
But it's not _so_ important for me.

Comment (dr0i) [reply conf/messages.de:15]: ah ... and it would be more in a tradition regarding "lobid.org" to call it Linked Open Data , not "linking data".

**Comment by fsteeg** [comment]
Result of our discussion:

1) We will replace "Open Data on the Web" with "Linked Open Data" in the header
2) We will link to http://json-ld.org/ from the API doc page and remove the "(JSON for Linking Data)"

**Pull request merged**